cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -318,7 +318,10 @@ export class Application {
     filterTags(source: string): void {
         this.applicationDetailsTab("Tags");
         cy.wait(2000);
-        cy.get(appDetailsView.tagFilter).click();
+        if (source != "Manual" && source != "Analysis")
+            cy.get(appDetailsView.tagCategoryFilter).click();
+        else cy.get(appDetailsView.tagFilter).click();
```

Comment:
It can be a little confusing to create an if without brackets and putting the code to execute in the next line but keeping the else in the same line.

I'd do one of two:

1. Condition and code to execute in same line
2. Condition and code to execute in different lines and code to execute surronded by brackets.

You can get more info at one of the most famous [JS/TS style guides](https://github.com/airbnb/javascript#blocks--braces)